CHANGES.rst

```diff
@@ -1,3 +1,7 @@
+1.5.2 (Unreleased)
+------------------
+- Apply correction to ACS TDD correction. [#100]
```

Comment:
I feel _"Apply correction to ... correction"_ is somewhat confusing. I think this should be formulated more explicitly.